src/GenAPI/Microsoft.DotNet.GenAPI/INamedTypeSymbolExtension.cs

```diff
@@ -195,41 +195,48 @@ public static bool HasInaccessibleTypeArgument(this INamedTypeSymbol namedType,
         /// <param name="symbolFilter">Assembly symbol filter <see cref="ISymbolFilter"/>.</param>
         public static IEnumerable<SyntaxNode> TryGetInternalDefaultConstructor(this INamedTypeSymbol namedType, ISymbolFilter symbolFilter)
         {
```

Comment:
note to reviewers: hide whitespace
<img width="324" alt="image" src="https://user-images.githubusercontent.com/8918108/235532582-79d98098-b8a9-4982-bb7d-d63328d5d63d.png">